src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -395,7 +399,8 @@ public IEnumerable<RowOptionDTO> GetAvailableRowsFor(ColumnMappingDTO model)
          }
 
          options.AddRange(excelColumns.Select(c => new RowOptionDTO() {Description = c, ImageIndex = ApplicationIcons.IconIndex(ApplicationIcons.ObservedDataForMolecule)}));
-         return options.OrderBy(o => o.ImageIndex).ThenBy(o => o.Description);
+         var metaDataIconIndex = ApplicationIcons.IconIndex(ApplicationIcons.ObservedDataForMolecule);
+         return options.OrderBy(o => o.ImageIndex == metaDataIconIndex).ThenBy(o => o.Description);
```

Comment:
I don't understand this? We are sorting by Icon??

Reply:
We want the excel columns (shown with metaDataIconIndex) to be listed at the end. Otherwise, by description alphabetically.

Reply:
So you are sorting by a boolean (FALSE, FALSE, TRUE, TRUE). ?Interesting